src/india_api/internal/inputs/indiadb/client.py

```diff
@@ -91,6 +91,16 @@ def get_predicted_power_production_for_location(
 
             # just select wind site
             sites = [s for s in sites if s.asset_type == asset_type]
+
+            # just select region
```

Comment:
Could this be a bit simpler by adding this condition to the previous list comprehension filter? Could then have the one line:

```
# just select asset type and region
sites = [s for s in sites if s.asset_type == asset_type and s.region == location]
```

Reply:
thats a good idea